docs/Displaying a User Interface/Template Images/index.md

```diff
@@ -8,6 +8,37 @@ You can easily display text, images, and buttons using the @![iOS]`SDLScreenMana
 | secondaryGraphic | The second image in a template that supports multiple images |
 
 ## Showing Images
+### Creating an SDLArtwork
+Create an `SDLArtwork` object which can be manually uploaded or set into the @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE]`ScreenManager`!@ and automatically uploaded. An SDLArtwork includes information about whether the image should be persisted between vehicle startups, whether the image is a template image and should be re-colored, and more.
+
+@![iOS]
+|~
+```objc
+//Create an SDLArtwork
+SDLArtwork* <#your variable name here#> = [SDLArtwork persistentArtworkWithImage:appImage name:@"<#Name to Upload As#>" asImageFormat:SDLArtworkImageFormatPNG /* or SDLArtworkImageFormatJPG */];
+```
+```swift
+//Create a UIImage 
+var <#variable for image#> = UIImage(named: <#Name to Upload As#>)!  
+//Create an SDLArtwork from the UIImage
+let <#variable for artwork#> = SDLArtwork(image: UIImage(named: <#variable for image#>, persistent: true, as: .PNG /* or .JPG */)
```

Comment:
1. The Obj-C version of the code doesn't match the Swift one (they have different formats). 
2. The `UIImage()` line here doesn't have an example of setting template image as previously asked.
3. These comments are obvious enough that they don't need to exist.
4. I'd recommend changing the `<#variable for x#>` part here to just `image` and `artwork` for line one and two respectively.
5. Line two (Swift) is incorrect, you're trying to insert a `UIImage` var into a string parameter of creating another `UIImage`. When you do (4), put `image`.
6. I previously mentioned that "Name to upload as" is not accurate. Change it to just "image name".
7. On the Obj-C code, the `*` goes next to the variable name, not the type.

Reply:
This still hasn't been addressed.